packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -346,4 +531,108 @@ contract DeployImplementations is Script {
         dsi_ = DeployImplementationsInput(DeployUtils.toIOAddress(msg.sender, "optimism.DeployImplementationsInput"));
         dso_ = DeployImplementationsOutput(DeployUtils.toIOAddress(msg.sender, "optimism.DeployImplementationsOutput"));
     }
+
+    function deployBytecode(bytes memory _bytecode, bytes32 _salt) public returns (address newContract_) {
+        assembly ("memory-safe") {
+            newContract_ := create2(0, add(_bytecode, 0x20), mload(_bytecode), _salt)
+        }
+        require(newContract_ != address(0), "DeployImplementations: create2 failed");
+    }
+}
+
+// Similar to how DeploySuperchain.s.sol contains a lot of comments to thoroughly document the script
+// architecture, this comment block documents how to update the deploy scripts to support new features.
+//
+// Using the base scripts and contracts (DeploySuperchain, DeployImplementations, DeployOPChain, and
+// the corresponding OPStackManager) deploys a standard chain. For nonstandard and in-development
+// features we need to modify some or all of those contracts, and we do that via inheritance. Using
+// interop as an example, they've made the following changes to L1 contracts:
+//   - `OptimismPortalInterop is OptimismPortal`: A different portal implementation is used, and
+//     it's ABI is the same.
+//   - `SystemConfigInterop is SystemConfig`: A different system config implementation is used, and
+//     it's initializer has a different signature. This signature is different because there is a
+//     new input parameter, the `dependencyManager`.
+//   - Because of the different system config initializer, we there is a new input parameter (dependencyManager)
+//
+// Similar to how inheritance was used to develop the new portal and system config contracts, we use
+// inheritance to modify up to all of the deployer contracts. For this interop example, what this
+// means is we need:
+//   - An `OPStackManagerInterop is OPStackManager` that knows how to encode the calldata for the
+//     new system config initializer.
+//   - A `DeployImplementationsInterop is DeployImplementations` that:
+//     - Deploys OptimismPortalInterop instead of OptimismPortal.
+//     - Deploys SystemConfigInterop instead of SystemConfig.
+//     - Deploys OPStackManagerInterop instead of OPStackManager, which contains the updated logic
+//       for encoding the SystemConfig initializer.
+//     - Updates the OPSM release setter logic to use the updated initializer.
+//  - A `DeployOPChainInterop is DeployOPChain` that allows the updated input parameter to be passed.
```

Comment:
Nope, it's not actually needed currently because of https://github.com/ethereum-optimism/optimism/pull/11623#discussion_r1752192921. So what happens is that DeployImplementations deploys `OPStackManagerInterop`, then you pass that address as input to `DeployOPChain`